src/Tools-CodeNavigation/CNSelectorExtractor.class.st

```diff
@@ -80,6 +80,11 @@ CNSelectorExtractor >> extractSelectorFromNode: aNode [
 		(node ~= originalNode and: [ 
 			 node parent notNil and: [ node parent isCascade ] ]) ifTrue: [ 
 			node := node parent messages first ] ].
+	
+	"In the case of doIt methods"
+	node selector = #noMethod & aNode isVariable
```

Comment:
not 
```Smalltalk
node selector = #noMethod and: [ aNode isVariable ]
```
?

Reply:
Looks like it matters very little in this case.
I can change it if you'd like =)